quadcontrol/config.c

```diff
@@ -487,6 +487,85 @@ int config_throttleRead(const char *path, quad_throttle_t **throttle, int *sz)
 }
 
 
+static int config_attenConverter(const hmap_t *h)
+{
+	int err = 0;
+	mma_atten_t *atten;
+
+	if (config_reallocData(cfg_attenuateID, sizeof(mma_atten_t)) != 0) {
+		return -1;
+	}
+
+	atten = (mma_atten_t *)res[cfg_attenuateID].data;
+
+	err |= config_parseFloat(h, "startVal", &atten->startVal);
+	err |= config_parseFloat(h, "endVal", &atten->endVal);
+	err |= config_parseFloat(h, "midVal", &atten->midVal);
+	err |= config_parseFloat(h, "midArg", &atten->midArg);
+
+	if (err != 0) {
+		return -1;
+	}
+
+	res[cfg_attenuateID].invCnt++;
+
+	return 0;
+}
+
+
+int config_attenRead(const char *path, mma_atten_t **atten, int *sz)
+{
+	int err;
+	parser_t *p;
+	static const unsigned int initSz = 1;
```

Comment:
```suggestion
	const unsigned int initSz = 1;
```
`static` is no needded

Reply:
Leaving it with static. All other parsers have this variable as `static const unsigned int` so that would be an issue to address in separate PR.